examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -356,7 +356,11 @@ void AppTask::StartThreadHandler(AppEvent * aEvent)
     if (aEvent->ButtonEvent.PinNo != THREAD_START_BUTTON)
         return;
 
-#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
+    if (AddTestPairing() != CHIP_NO_ERROR)
```

Comment:
This is confusing, so by default we're adding a test pairing?

Reply:
This code executes when you press **Button 3**. Previously the test pairing was added automatically on the application startup. **Button 3** triggers a test mode. That is, it starts Thread with default settings so it makes sense to add default CHIP keys then.